### ISSUE TITLE
Control tag masking option

### ISSUE THREAD

**Issue by garykbrixi** [Zymrael/vortex#33]
To support sequence scoring we should add an option to mask control tag for pll calculation so they are excluded from logits/seq likelihood
